dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -297,6 +298,66 @@ public Organization updateOrganizationDescription(@ApiParam(hidden = true) @Auth
         return eventDAO.findEventsForOrganization(id, offset, limit);
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @Path("/{organizationId}/star")
```

Comment:
I know we haven't really been following REST principles so far.  But if we were to start doing so, avoid using verbs in the endpoint.  In this case, "star"  and "unstar" appears to be a verb.

`StarRequest request` is not used but needed in the PUT.  It's a sign that it's not REST.

Reply:
Charles was explaining RESTful practices to me and proposed the idea of having only a PUT, and using the StarRequest body to determine whether to star/unstar it. And then /star could be thought more of as a noun? Thoughts?

Reply:
That sounds better

Reply:
ga4gh/dockstore#2395